source/template/introduction.rst

```diff
@@ -1,7 +1,7 @@
 简介
 ====
 
-本文档是 **地震“学”软件** 的教程模板。该目标规定了教程中所需包含的内容，
+本文档是\ **地震“学”软件**\ 的教程模板。该模板规定了软件教程中所需包含的内容，
```

Comment:
Why sometimes you add `\` before and after the whitespace?

Reply:
Short answer: to remove whitespace between Chinese characters.

Without `\ `:

![image](https://user-images.githubusercontent.com/3974108/103473833-d9bf5800-4d6a-11eb-9110-71ca88aa1526.png)

With `\ `:
![image](https://user-images.githubusercontent.com/3974108/103473839-e479ed00-4d6a-11eb-921e-db6d0750e467.png)